modules/runtime/nuxeo-stream/src/test/java/org/nuxeo/lib/stream/tests/log/TestLogChronicle.java

```diff
@@ -135,6 +131,8 @@ public void testFileRetention() throws Exception {
     @SuppressWarnings("FutureReturnValueIgnored")
     @Test
     public void testConcurrentFileRetentions() throws Exception {
+        org.junit.Assume.assumeFalse(IS_WIN);
```

Comment:
I was expecting this test to pass to prove that the bug is fixed, we need a reason on assumeFalse